_translator-files/po/de/Include-QOS-Windows.po

```diff
@@ -3,49 +3,50 @@
 # This file is distributed under the same license as the PACKAGE package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
 #
-#, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"Project-Id-Version: \n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2022-05-16 17:41+0200\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: de\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.0.1\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-QOS-Windows.md:3
 msgid "[comment]: # (This is an include file for use in multiple documents)"
-msgstr ""
+msgstr "[Kommentar]: # (Dies ist eine Include-Datei zur Verwendung in mehreren Dokumenten)"
 
 #. type: Plain text
 #: ../wiki/en/Include-QOS-Windows.md:5
 msgid "# Quality of Service (QoS)"
-msgstr ""
+msgstr "# Quality of Service (QoS)"
 
 #. type: Plain text
 #: ../wiki/en/Include-QOS-Windows.md:7
 msgid "Jamulus uses DSCP/CS4 opportunistically to deal with buffer bloat, and uses a default DSCP/CS4 value of 128 (or 0x80). This is compatible with IPv4 and IPv6. Other values can be set using the -Q option, eg -Q [0..255] (where 0 disables QoS). If you want to explore the effect of non-default settings, see RFC4594. However, most people will have no need to do this."
-msgstr ""
+msgstr "Jamulus verwendet DSCP/CS4 opportunistisch, um mit Bufferbloat umzugehen, und verwendet einen Standard-DSCP/CS4-Wert von 128 (oder 0x80). Dieser Wert ist mit IPv4 und IPv6 kompatibel. Andere Werte können mit der Option -Q festgelegt werden, z. B. -Q [0..255] (wobei 0 die QoS deaktiviert). Wenn du die Auswirkungen von nicht standardmäßigen Einstellungen untersuchen möchtest, lese die RFC4594. Die meisten Benutzer werden dies jedoch nicht benötigen."
```

Comment:
Sounds very technical...Probably worth another issue though. 

The RFC should be linked...